.github/workflows/ci-test-memory-growth.yml

```diff
@@ -1,4 +1,4 @@
-name: SMARTS CI Master
+name: SMARTS CI Memory
```

Comment:
Perhaps this one should also be enabled and run on schedule.

Reply:
I have enabled the memory test to run on schedule.